phangsPipelineTests/test_casaCubeRoutines.py

```diff
@@ -0,0 +1,173 @@
+"""
+How to run this test inside CASA:
```

Comment:
Awesome! We should setup CI to start running the tests in a future PR.

Reply:
Done